python/lsst/ts/tunablelaser/hardware.py

```diff
@@ -492,7 +487,7 @@ def __init__(self, commander, simulation_mode=False):
                 module_id=self.id,
                 register_name="Configuration",
                 read_only=False,
-                accepted_values=["No SCU", "F1 No SCU", "F2 No SCU"],
+                accepted_values=list(NoSCU),
```

Comment:
All these if/else conditionals, starting at line 483, should use you new enum class.

(I can't seem to get github to let me move this up where it belongs, just after line 483)